pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1467,6 +1467,27 @@ class DateOffset(RelativeDeltaOffset, metaclass=OffsetMeta):
         previous midnight.
     **kwds
         Temporal parameter that add to or replace the offset value.
+    weekday : int {0, 1, ..., 6}, default 0
```

Comment:
can this go before `kwds`? that should typically be the last argument

Reply:
Done.